sw/device/lib/testing/test_framework/ottf_main.h

```diff
@@ -105,15 +106,20 @@ void ottf_task_delete_self(void);
 char *ottf_task_get_self_name(void);
 
 /**
- * Execute a test function and log the test result.  Update the result value
- * if there is a failure code.
+ * Execute a test function, profile the execution and log the test result.
+ * Update the result value if there is a failure code.
  */
 #define EXECUTE_TEST(result_, test_function_)                            \
   do {                                                                   \
     LOG_INFO("Starting test " #test_function_ "...");                    \
+    uint64_t t_start_ = ibex_mcycle_read();                              \
     status_t local_status = INTO_STATUS(test_function_());               \
+    uint32_t cycles_ = ibex_mcycle_read() - t_start_;                    \
+    uint32_t micros = udiv64_slow(cycles_, kClockFreqCpuHz, NULL);       \
```

Comment:
There seems to be a factor of 1000000 missing from this computation (e.g. `(uint64_t)cycles * 1000000`)

Reply:
Good catch, I was using `kClockFreqCpuMhz`, but I had to refactor it because this constant is not available for Verilator (as its clock is 500 KHz), and this introduced this error.

Fixed.